tests/pytests/integration/cli/test_salt_key.py

```diff
@@ -172,6 +172,8 @@ def test_list_all_no_check_files(
     shutil.copytree(salt_master.config["pki_dir"], str(pki_dir))
     with pytest.helpers.change_cwd(str(config_dir)):
         master_config = copy.deepcopy(salt_master.config)
+        # Convert the file roots to a static dict before appending any roots
+        master_config["file_roots"] = master_config["file_roots"].static_dict()
```

Comment:
Maybe try skipping this line on Windows?
```
if not salt.utils.platform.is_windows():
```

Reply:
I added that to attempt to fix it as that was the only thing I could see affecting it. It didn't make a difference either way though.